packages/amplify_datastore/example/lib/models/SimpleCustomType.dart

```diff
@@ -31,10 +31,10 @@ class SimpleCustomType {
     try {
       return _foo!;
     } catch (e) {
-      throw DataStoreException(
-          DataStoreExceptionMessages
+      throw AmplifyCodeGenModelException(
```

Comment:
This is a breaking change we should note

Reply:
Hum I missed this part somehow, I don't this this is an expected change, let me double check, could be just the test model is outdated (generated with WIP codegen of other features)

Reply:
I believe this is expected change in codegen that is specific to > 0.4.0, though possible we did not note this change properly. However, I think any breakage was already with 0.3.x - 0.4.x